tfjs-core/src/ops/browser.ts

```diff
@@ -299,25 +325,10 @@ export async function toPixels(
     $img = cast(originalImgTensor, 'int32');
     originalImgTensor.dispose();
   }
-  if ($img.rank !== 2 && $img.rank !== 3) {
-    throw new Error(
-        `toPixels only supports rank 2 or 3 tensors, got rank ${$img.rank}.`);
-  }
+  validateImgTensor($img);
```

Comment:
Maybe add a warning in this API to tell users that this API will be deprecated and suggest them switch to the more powerful `draw` API.

Reply:
Done. Thanks!